homeassistant/components/compensation/const.py

```diff
@@ -4,6 +4,8 @@
 SENSOR = "compensation"
 
 CONF_COMPENSATION = "compensation"
+CONF_APPLY_LOWER_LIMIT = "apply_lower_limit"
```

Comment:
I think if this as ‘clamp’ instead of ‘apply’ but apply may make more sense to someone else so I other way works

Reply:
Done